projects/ngx-vest-forms/src/lib/directives/form-model-group.directive.ts

```diff
@@ -26,13 +27,15 @@ import { getFormGroupField } from '../utils/form-utils';
 })
 export class FormModelGroupDirective implements AsyncValidator {
   private readonly formDirective = inject(FormDirective);
+  private readonly validationOptionsDirective = inject(ValidationOptionsDirective, { self: false, optional: true });
```

Comment:
Shouldn't this be set to true? Will the form model group inject its own validationOptionsDirective or one from its ancestors?

Reply:
I replaced the inject with an input. This makes it even cleaner in my opinion.